modules/electromagnetics/doc/content/source/auxkernels/CurrentDensity.md

```diff
@@ -31,6 +31,13 @@ MOOSE errors will be thrown if the inappropriate coupled variable is provided,
 given the setting for the [!param](/AuxKernels/CurrentDensity/electrostatic) 
 boolean parameter.
 
+Additionally, note that the electrical conductivity $\sigma$ must be provided
+as a material property for the block(s) this kernel is being applied to. The
+user can use, e.g., `GenericConstantMaterial`, `GenericFunctionMaterial`,
+`GenericFunctorMaterial`, or one of their AD counterparts. The user must also
```

Comment:
```suggestion
user can use, e.g., [GenericConstantMaterial.md], [GenericFunctionMaterial.md], or one of their AD counterparts. The user must also
```

the `GenericFunctorMaterial` creates functor material properties. they are very different and dont work as material properties

Reply:
I'm glad you checked this, because I was actually guessing when I wrote that.
Do you plug them differently into the sim?

Reply:
yeah you do. They are consumed using getFunctor not getMaterialProperty